lib/trino-collect/src/main/java/io/trino/collect/cache/EmptyCache.java

```diff
@@ -116,7 +117,7 @@ public ConcurrentMap<K, V> asMap()
             public V putIfAbsent(K key, V value)
             {
                 // Cache, even if configured to evict everything immediately, should allow writes.
-                return value;
+                return null;
```

Comment:
good catch

Can you move it to a separate PR?

Reply:
https://github.com/trinodb/trino/pull/14702